docs/pages/index.mdx

```diff
@@ -4,7 +4,7 @@ import "../styles.css";
 <CustomHomePage.Root>
   <CustomHomePage.Headline>Build with XMTP</CustomHomePage.Headline>
   <CustomHomePage.Subhead>
-    XMTP is an open, secure, and resilient messaging protocol—for the next phase of the internet
+    XMTP is a secure and decentralized protocol for building communication platforms for the next phase of the internet.
```

Comment:
Is this language we have gone over with GTM? We should have a consistent one-sentence "XMTP is..."

Reply:
@saulmc thank you for this callout and agreed! I got this from one of Shane's exploratory docs. I will surface this topic with GTM so we can align.